tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -19,6 +19,16 @@
 	<!-- setup classpath -->
 	<path id="project.classpath" />
 
+	<!-- Provide the JUnit Ant tasks even when optional Ant libraries (ant-junit.jar) are not installed system-wide. -->
```

Comment:
Changes to `customBuild.xml` do not belong in this PR.
Also, please clarify under which circumstances these libraries are missing, since they should be present when the IDE is set up according to the instructions at: https://github.com/tlaplus/tlaplus/tree/master/general/ide

Reply:
My local Ant didn't have the optional ant-junit/ant-optional tasks, so I bundled  the JARs to run `ant -f tlatools/org.lamport.tlatools/customBuild.xml` test. I've reverted those build changes to keep this PR focused on the Vector  refactor